src/crypto/CHIPCryptoPAL.h

```diff
@@ -630,6 +630,11 @@ class P256Keypair : public P256KeypairBase
     /** Release resources associated with this key pair */
     void Clear();
 
+#if CHIP_CRYPTO_TRUSTY_OS
+    uint64_t mP256Handler = 0;
```

Comment:
why are these public?

Who sets mFabricIndex? it seems initialized to 0 and never set but only used. What does "mFabricIndex" actually mean and why is it relevant in a crypto PAL (needs comments). Also what is a "Handler".

I believe these need more comments overall.